src/main/kotlin/io/snyk/plugin/net/SnykApiClient.kt

```diff
@@ -8,42 +8,36 @@ import retrofit2.Retrofit
  * Main entrypoint for using Snyk API.
  */
 class SnykApiClient constructor(
-    private val retrofit: Retrofit
+    retrofit: Retrofit
```

Comment:
with a retrofit mock, testing the ApiClient should be possible & straightforward

Reply:
done